ckanext/canada/templates/scheming/display_snippets/dataset_relationship.html

```diff
@@ -0,0 +1,17 @@
+{% set fields = data[field.field_name] %}
+
+{% block subfield_display %}
+  <div>
+    {%- for field_data in fields -%}
+      <p><i class="fa fa-external-link" aria-hidden="true"></i>&nbsp;
+        {%- for subfield in field.repeating_subfields -%}
+          {%- snippet 'scheming/snippets/display_field.html',
+            field=subfield,
+            data=field_data,
+            entity_type=entity_type,
+            object_type=object_type -%}
+        {%- endfor -%}
+      </p>
+    {%- endfor -%}
+  </div>
+{% endblock %}
```

Comment:
shouldn't this be automatic, or is there a modification to the default here?

Reply:
thanks, maybe add a comment about why it's required and when we might be able to remove it

Reply:
comments added.